src/styles/custom.css

```diff
@@ -450,32 +450,30 @@ li[style*="--depth: 1"] {
   @apply pl-3 border-l-[1px] border-kinde-grey-100 dark:border-kinde-grey-800;
 }
 
-starlight-tabs[data-sync-key="starlight-package-managers-pkg"] {
-  @apply pt-3;
+starlight-tabs {
+  @apply pt-3 after:absolute relative after:top-0 after:right-0 after:w-12 after:from-white after:to-transparent block after:content-[""] after:bg-gradient-to-l after:h-12 dark:after:from-black;
```

Comment:
Not applicable - tab height won't change, items are still visible in case of many items and the use of gradient in our styles are not common, so this is a one-off style.